cvat-ui/src/components/analytics-page/task-quality/quality-settings-form.tsx

```diff
@@ -21,7 +24,16 @@ interface FormProps {
 export default function QualitySettingsForm(props: FormProps): JSX.Element | null {
     const { form, settings } = props;
 
+    const schemaDescriptions = useSelector((state: CombinedState) => (
+        state.serverAPI.schema?.components.schemas.QualitySettings.properties
```

Comment:
Probably it would be better to put it to `QualitySettings` class as `descriptions` property, which is a dictionary

Reply:
But its two separate endpoints, we wil need to do something like `settings.descriptions = descriptions` each time we get/update settings. 
We cant do it from cvat-core part initially. We can make some kind of injection from ui after getting schema so settings will auto patched, but it looks too complicated for just a tooltip

Reply:
> But its two separate endpoints, we wil need to do something like settings.descriptions = descriptions each time we get/update settings 

We know that schema, generally, not changed => we do not need to re-fetch server schema each time. I am not sure what problem with initialization you want to emphasize.

Reply:
> We cant do it from cvat-core part initially.

Why we can't do it from cvat-core part initially? The class is initialized by asyncly fetched data. What is the problem to put more data to the constructor?

Reply:
When we are getting data somewhere in [api implementation](https://github.com/cvat-ai/cvat/blob/develop/cvat-core/src/api-implementation.ts#L516). We dont know anything about schema. So we either need to fetch it from server, or get it as a param from cvat-ui. Both approaches are not good.
Or you're suggesting to create something like cache for the schema somewhere in core part initially and then use it in api-implementations?

Reply:
> to fetch it from server

Exactly this when schema is not fetched yet. If it was fetched, we may use schema from cache. 
Cache may be created anywhere in cvat-core, e.g. in server-proxy.

Reply:
`cvat-ui` ideally does not now anything about server and in the most of cases it works with cvat-core. 
That is not possible to achieve in plugins when core itself does not now about optional endpoint, but that is general approach

Reply:
Done. Used separate file, not server-proxy as its already quite big